asdf/tests/test_api.py

```diff
@@ -352,38 +354,36 @@ def test_extension_version_check(installed, extension, warns):
 
 def test_auto_inline(tmpdir):
     outfile = str(tmpdir.join('test.asdf'))
-    tree = {"small_array": np.arange(6), "large_array": np.arange(100)}
+    small = 75
+    large = constants.DEFAULT_AUTO_INLINE
+    tree = {"small_array": np.arange(small), "large_array": np.arange(large)}
 
     # Use the same object for each write in order to make sure that there
     # aren't unanticipated side effects
     with asdf.AsdfFile(tree) as af:
         # By default blocks are written internal.
         af.write_to(outfile)
-        assert len(list(af.blocks.inline_blocks)) == 0
-        assert len(list(af.blocks.internal_blocks)) == 2
-
-        af.write_to(outfile, auto_inline=10)
         assert len(list(af.blocks.inline_blocks)) == 1
         assert len(list(af.blocks.internal_blocks)) == 1
 
-        # The previous write modified the small array block's storage
-        # to inline, and a subsequent write should maintain that setting.
-        af.write_to(outfile)
-        assert len(list(af.blocks.inline_blocks)) == 1
-        assert len(list(af.blocks.internal_blocks)) == 1
+        af.write_to(outfile, auto_inline=10)
+        assert len(list(af.blocks.inline_blocks)) == 0
+        assert len(list(af.blocks.internal_blocks)) == 2
 
-        af.write_to(outfile, auto_inline=7)
+        # The previous write modified the small array block's storage to
+        # inline, and a subsequent write should not maintain that setting.
+        af.write_to(outfile)
         assert len(list(af.blocks.inline_blocks)) == 1
         assert len(list(af.blocks.internal_blocks)) == 1
 
-        af.write_to(outfile, auto_inline=5)
-        assert len(list(af.blocks.inline_blocks)) == 0
-        assert len(list(af.blocks.internal_blocks)) == 2
+        af.write_to(outfile, auto_inline=150)
```

Comment:
... and this relative to `large`

Reply:
I changed this to be relative to `large`.